app/models/school_validator.rb

```diff
@@ -133,8 +140,7 @@ class SchoolValidator
   validates :alternate_last_name, if: :has_alternate?, presence: true
   validates :alternate_shirt_size, if: :has_alternate?, presence: true
 
-  def initialize(attributes={}, page=0)
-    @page = page
+  def initialize(attributes={})
```

Comment:
Since we pass in `current_page` with the `update_params`, we don't need to store it!